resources/south-america/brazil/mapeadoresYouthMappersUFRJ.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "OSM-group",
+  "type": "group",
+  "locationSet": {
+  
+    "include": ["076"];
+    "include": ["005"];
+    "include": ["rio_de_janeiro.geojson"];
+  
+  },
+  "languageCodes": ["pt"],
+  "order": 4,
+  "strings": {
+    "community": "YouthMappers UFRJ",
+    "comunityID": "mapeadoresYouthMappersUFRJ",
+    "description": "All are welcome! To see details, please, visit {signupUrl}",
+    "signupUrl": "https://ivides.org/youthmappers-ufrj",
+    "url": "https://tasks.teachosm.org/users/YouthMappers"
```

Comment:
this gives login wall if you have no third-party account

also, it seems to be YouthMappers link, not YouthMappers UFRJ link judging by URL